mailpoet/lib/Automation/Engine/Builder/UpdateWorkflowController.php

```diff
@@ -40,6 +46,12 @@ public function updateWorkflow(int $id, array $data): Workflow {
     return $workflow;
   }
 
+  private function checkWorkflowName(string $name): void {
```

Comment:
I think you can just use `Builder::string()->minLength(1)` [here](https://github.com/mailpoet/mailpoet/blob/3bbb662857959307fd895fdabb61bdf89a890f15/mailpoet/lib/Automation/Engine/Endpoints/Workflows/WorkflowsPutEndpoint.php#L34).

Reply:
Done.